apps/prairielearn/elements/pl-multiple-choice/pl-multiple-choice.mustache

```diff
@@ -154,7 +158,7 @@
 {{/submission}}
 {{#answer}}
     {{#radio}}
-        {{#inline}}<span class="d-inline-block"><ul class="list-inline mb-0">{{/inline}}
+        {{#inline}}<fieldset class="d-inline-block"><legend>Multiple choice options</legend><ul class="list-inline mb-0">{{/inline}}
```

Comment:
Why a `fieldset` here? There aren't any inputs in the answer panel.

Reply:
Reverted in [6f54747](https://github.com/PrairieLearn/PrairieLearn/pull/10514/commits/6f547473f9ac91f949f5e3f77ced434e15291109).